master/buildbot/newsfragments/thread_safety_of_dockerlatentworker.bugfix

```diff
@@ -0,0 +1 @@
+The function _thd_start_instance in :py:class:`~buildbot.plugins.worker.DockerLatentWorker` uses now thread-local variables to hold the binds and volumes arguments. Previously member variables were used which is not thread-safe.
```

Comment:
This is too much details.

Just use: fix threading issue in DockerLatentWorker

The details belongs to the commit messages